src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,88 @@
+package duke;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * <code>duke.Storage</code> handles all file read and writes.
+ */
+public class Storage {
+    final static String TASKS_TEXT_FILE_PATH = "../../tasks.txt";
```

Comment:
Should there be an access modifier? I noticed the same issue in several other places too.

Reply:
You are right! I forgot about them. Thanks for highlighting it.